marco-kavdi/starter-code/scripts/articleView.js

```diff
@@ -55,15 +55,28 @@ articleView.handleAuthorFilter = function() {
 };
 
 articleView.handleCategoryFilter = function() {
-  // TODO: Just like we do for #author-filter above, we should handle change events on the #category-filter element.
+  $('#category-filter').on('change', function() {
+    if ($(this).val()){
+      $('article').hide();
+      $('article[data-category="' + ($(this).val()) + '"]').fadeIn();
+    }
+    else {
+      $('article:not(.template)').fadeIn();
+    }
+  })
+  // TODO:DONE Just like we do for #author-filter above, we should handle change events on the #category-filter element.
   //       When an option with a value is selected, hide all the articles, then reveal the matches.
   //       When the blank (default) option is selected, show all the articles, except for the template.
   //       Be sure to reset the #author-filter while you are at it!
 
 };
 
 articleView.handleMainNav = function() {
-  // TODO: Add an event handler to .main-nav elements that will power the Tabs feature.
+  $('.main-nav .tab').on('click', function(){
+    $('.tab-content').hide();
+    $('.tab-content[id="' + $(this).attr('data-content') + '"]').fadeIn();
```

Comment:
Since the selector of an id is just `#the-id` you do not need such a complicated selector here.